### DISCUSSION TITLE
Ständiger Zeilensprung auf Android und Apple Tabletts in Landscape Position

### DISCUSSION THREAD

**Discussion by T2023om** [tbnobody/OpenDTU#1453] [Ideas]
Beim Hochzählen der Sekunden erfolgt bei Sekunde 1 der Sprung auf einzeilig und bei Sekunde 2 der Sprung auf zweizeilig.
Das passiert bei jedem Abfrageintervall erneut und ist das Ergebnis der unterschiedlichen Ziffernbreite.

Anbei zwei 2 Bilder auf einem iPad, die den Sachverhalt zeigen und das macht keinen Spaß da zuzusehen.

![Bild_1_Sekunde](https://github.com/tbnobody/OpenDTU/assets/145997427/2550243a-c7c1-43aa-9a85-c9dd69c38170)
![Bild_3_Sekunden](https://github.com/tbnobody/OpenDTU/assets/145997427/acadf7fa-4058-4e05-8ae4-f85fe2d7b5ac)

Bei 4 Button in der Zeile passiert die ständige Umschaltung noch nicht, aber bei 5 ist scheinbar mindestens ein Pixel in der Zeile zu wenig.

Um das Problem zu lösen, schlage ich eine Änderung der app.js vor.
In der:  div class="d-flex flex-wrap"

von:   Yy={style:{"padding-right":"**2em**"}     
auf:    Yy={style:{"padding-right":"**1em**"}

Meine Versuche haben keine negativen Auswirkungen auf die Anzeige ergeben und vielleicht hilft es ja auch anderen Nutzern,
da ich mir nicht vorstellen kann das dieses Verhalten nur bei mir auftritt.

**Comment by Tiese** [comment]
Hallo T2023om,

klasse - jetzt "zuckt" nichts mehr :-)

Vielen Dank!


Schöne Grüße
Tiese

**Reply by T2023om** [reply]
Hallo Tiese,

danke für die Blumen. es freut mich das es auch bei dir genau so geklappt hat.

Idee:
Willst du nicht deine aktuelle Firmware mit der Änderung updaten?

Lg. 
T2023om

**Reply by Tiese** [reply]
Guten Morgen T2023om,

die Änderung "VERBESSERUNG" sind schon eingeflossen!
Ich will es nur noch nicht hochladen, weil ich einen Email-Benachrichtigungsdienst eingefügt habe und diesen erst ausgiebig testen will!!!

**Comment by tbnobody** [comment]
Die Änderung siehe #1440 hat nicht geholfen?

**Reply by Tiese** [reply]
@tbnobody: Auf jeden Fall ist es jetzt besser, als vorher!
